HaxmLoader/haxm_loader.c

```diff
@@ -32,8 +32,32 @@
 #include <strsafe.h>
 #include <windows.h>
 
+#define WIDE2(x) L##x
+#define WIDE1(x) WIDE2(x)
+#define WFUNCTION WIDE1(__FUNCTION__)
+
+#define SERVICE_NAME_RELEASE L"IntelHaxm"
 #define SERVICE_NAME L"haxm service for test"
```

Comment:
Do you think it is appropriate to rename the SERVICE_NAME to "IntelHaxm_t" or "HaxmTest" as it is inconvenient for command line parameter using the long string with spaces?

Reply:
emm, good point. I'll change that too.